packages/odie-client/src/types.ts

```diff
@@ -77,8 +77,13 @@ export type Source = {
 		ui_position: number;
 		ui_algo: string;
 		fetch_algo: string;
+		fetch_lang: string;
 		fetch_position: number;
+		fetch_query: number;
 		railcar: string;
+		rec_blog_id: string;
```

Comment:
We should have here the rec_query property, as string.

Also, while we are here in Calypso we tend to order the typings alphabetically, not important but ease when looking for a discrete value.

Reply:
I realized we are not using `rec_query`, so no need to add it here. I did re-sort the list alphabetically though.